diff --git a/src/Drupal/Bootstrap.php b/src/Drupal/Bootstrap.php
index 0be5f7b..bb1855a 100644
--- a/src/Drupal/Bootstrap.php
+++ b/src/Drupal/Bootstrap.php
@@ -165,6 +165,7 @@ class Bootstrap
         $core_tests_dir = $this->drupalRoot . '/core/tests/Drupal';
         $this->namespaces['Drupal\\Tests'] = $core_tests_dir . '/Tests';
         $this->namespaces['Drupal\\TestSite'] = $core_tests_dir . '/TestSite';
+        $this->namespaces['Drupal\\TestTools\\PhpUnitCompatibility'] = $core_tests_dir . '/TestTools/PhpUnitCompatibility';
         $this->namespaces['Drupal\\KernelTests'] = $core_tests_dir . '/KernelTests';
         $this->namespaces['Drupal\\FunctionalTests'] =  $core_tests_dir . '/FunctionalTests';
         $this->namespaces['Drupal\\FunctionalJavascriptTests'] = $core_tests_dir . '/FunctionalJavascriptTests';
